docs/tutorials/model_to_instance.ipynb

```diff
@@ -0,0 +1,314 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import jijmodeling as jm\n",
+    "\n",
+    "# アイテムの価値\n",
+    "v = jm.Placeholder(\"v\", ndim=1)\n",
+    "# アイテムの重さ\n",
+    "w = jm.Placeholder(\"w\", ndim=1)\n",
+    "# ナップサックの耐荷重\n",
+    "W = jm.Placeholder(\"W\")\n",
+    "# アイテムの総数\n",
+    "N = v.len_at(0, latex=\"N\")\n",
+    "# アイテムiをナップサックに入れる場合は1, 入れない場合は0を取る決定変数\n",
+    "x = jm.BinaryVar(\"x\", shape=(N,)) \n",
+    "# アイテムに割り当てられた番号を走る添え字\n",
+    "i = jm.Element(\"i\", belong_to=(0, N))\n",
+    "\n",
+    "problem = jm.Problem(\"problem\", sense=jm.ProblemSense.MAXIMIZE)\n",
+    "# 目的関数\n",
+    "problem += jm.sum(i, v[i] * x[i])\n",
+    "# 制約条件: ナップサックの耐荷重を超えない\n",
+    "problem += jm.Constraint(\"重量制限\", jm.sum(i, w[i] * x[i]) <= W)\n",
+    "problem"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step2. Interpreter オブジェクトにインスタンスデータを登録する\n",
+    "\n",
+    "Step1で定式化した数理モデルの係数に入力するデータ（インスタンスデータ）を用意します。これは `Interpreter` オブジェクトを使用して実現できます。\n",
+    "\n",
+    "`Interpreter` クラスのコンストラクタの引数に、以下のペアを持つ辞書を渡すことでインスタンスデータを登録できます：\n",
+    "\n",
+    "- キー：`Placeholder` オブジェクトの `name` プロパティに設定した文字列\n",
+    "- 値：入力するデータ"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "interpreter = jm.Interpreter({\n",
+    "    \"v\": [10, 13, 18, 31, 7, 15],  # アイテムの価値のデータ\n",
+    "    \"w\": [11, 15, 20, 35, 10, 33], # アイテムの重さのデータ\n",
+    "    \"W\": 47,                       # ナップサックの耐荷重のデータ\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step3. Interpreter オブジェクトを使って数理モデルをインスタンスに変換する\n",
+    "\n",
+    "数理モデルをインスタンスデータに変換するには、Interpreter.eval_problem メソッドを使用します。インスタンスデータが登録された Interpreter オブジェクトの eval_problem メソッドに Problem オブジェクトを渡すと、その Problem オブジェクトが持つ Placeholder にデータを入力し、インスタンスに変換して ommx.v1.Instance オブジェクトとして返します："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "instance = interpreter.eval_problem(problem)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## OMMX Adapterに入力して最適化問題を解く\n",
+    "\n",
+    "Step3で得られた `ommx.v1.Instance` オブジェクトをOMMX Adapterに入力することで、最適化計算を実行できます。\n",
+    "例えば、OMMX Adapterの一つである `ommx-python-mip-adapter` ([GitHub](https://github.com/Jij-Inc/ommx/tree/main/python/ommx-python-mip-adapter), [PyPI](https://pypi.org/project/ommx-python-mip-adapter/)) を使用して、そのインスタンスの最適解と最適値を求めるPythonコードは次の通りです："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from ommx_python_mip_adapter import instance_to_model, model_to_solution\n",
+    "\n",
+    "# ommx.v1.instanceオブジェクトからPython-MIPのModelオブジェクトに変換する\n",
+    "model = instance_to_model(instance)\n",
+    "# Python-MIPのデフォルトソルバー (SCIP) で最適化を実行する\n",
```

Comment:
```suggestion
    "# Python-MIPのデフォルトソルバー (CBC) で最適化を実行する\n",
```

Reply:
fix in https://github.com/Jij-Inc/JijModeling-Tutorials/pull/6/commits/cb9142ec00c6ad89abf8381d8fc053b9327c6f15